src/store.ts

```diff
@@ -1,7 +1,7 @@
 import { atom, StoreValue } from 'nanostores'
 
 export const isNavigationOpen = atom<boolean>(false)
-export const isFundraiserOpen = atom<boolean>(true)
+export const isFundraiserOpen = atom<boolean>(false)
```

Comment:
This change is irrelevant to the #115  but boy-scouting the minor layout issue I noticed.

![image](https://github.com/user-attachments/assets/d7f3dbcd-af4a-455c-88d2-b5df207b7b9c)